x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```diff
@@ -578,6 +582,10 @@ processors:
     field: checkpoint.industry_reference
     target_field: vulnerability.id
     ignore_missing: true
+- date:
+    field: "checkpoint.time"
+    formats: ["ISO8601", "UNIX"]
+    if: "ctx.checkpoint?.time != null"
 - rename:
```

Comment:
I would delete this rename block, since the previous `date` one will take care of this